app/frontend/src/features/map/MapPage.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import PageTitle from "../../components/PageTitle";
+import Map from "../../components/Map";
+import { userRoute } from "../../AppRoutes";
+import { LngLat, Map as MapboxMap } from "mapbox-gl";
+import { useHistory, useLocation } from "react-router-dom";
+
+import { addClusteredUsersToMap } from "./ClusteredUsers";
+
+export default function MapPage() {
+  const history = useHistory();
+
+  const location = useLocation();
+
+  const userClicked = (ev: any) => {
```

Comment:
```suggestion
  const handleClick = (ev: any) => {
```
Makes it a bit clearer it's an event handler callback based off some event/action.